### PR TITLE
Visualizations for regression

### PR DESCRIPTION
**Pull request by anuprulez**
This pull request adds three plots for regression tasks: true vs predicted curves, scatter plots for true and predicted targets and a residual plot. It takes two tabular input files: true and predicted targets.

True vs predicted curves:

true_pred_curves

Scatter plot for true and predicted targets:

true_vs_pred_scatter

Residual plot:

residual_plot

### CODE REVIEW

**Comment by anuprulez** [comment]
ping @bgruening @qiagu

**Comment by qiagu** [comment]
👍 Great

File: tools/plotly_regression_performance_plots/plotly_regression_performance_plots.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    <version_command>echo $version</version_command>
+    <command detect_errors="aggressive"><![CDATA[
+    python '$__tool_directory__/plot_regression_performance.py'
+        -i '$infile_input'
+        -j '$infile_output'
+]]>
+    </command>
+    <inputs>
+        <param name="infile_input" type="data" format="tabular" label="Select input data file :" help="Input data is a matrix (tabular) where each column is a feature and the last column contains the target values."/>
+        <param name="infile_output" type="data" format="tabular" label="Select predicted data file :" help="Predicted data is a matrix (tabular) where each column is a feature and the last column contains the predicted values."/>
```
Comment (qiagu) [inline tools/plotly_regression_performance_plots/plotly_regression_performance_plots.xml]: I think once the last column contains the designated values, the data can be retrieved for visualization. In other words, files contain one column which are the true labels are still valid input here.

Comment (anuprulez) [reply tools/plotly_regression_performance_plots/plotly_regression_performance_plots.xml]: Yes, just true and predicted value columns are enough for inputs. But, we just get complete data either with true values or predicted values.

Comment (qiagu) [reply tools/plotly_regression_performance_plots/plotly_regression_performance_plots.xml]: How about this?
```A tabular in which the last column (including the column if there is only one) contains the target values ```

Comment (anuprulez) [reply tools/plotly_regression_performance_plots/plotly_regression_performance_plots.xml]: Update the help tex. Thanks :)

**Comment by bgruening** [comment]
Thanks both! We do have a tool installation stop until we finish migration of usegalaxy.eu to the new head node. Sorry for that.

**Pull request merged**